Source/DafnyCore/Resolver/DetectUnsoundFunctionReferences_Visitor.cs

```diff
@@ -0,0 +1,37 @@
+using System.Diagnostics.Contracts;
+
+namespace Microsoft.Dafny;
+
+class DetectUnsoundFunctionReferences_Visitor : ResolverBottomUpVisitor {
```

Comment:
Why the `_` in the name? I think that's outside of C# conventions. Does Rider not suggest to remove it?

Reply:
I had followed the convention used by other `_Visitor` classes in the Dafny sources. But I agree that's outside C# conventions. So, I now renamed all classes in Dafny to remove `_`'s.